.travis.yml

```diff
@@ -4,3 +4,8 @@ node_js:
   - "5"
 git:
   depth: 10
+
+branches:
```

Comment:
Hmm, does this disable test runs for pull-requests? I've always liked having that functionality.